mimesis/providers/numbers.py

```diff
@@ -1,14 +1,15 @@
-import array
-import mimesis.typing as types
+from mimesis import typing as types
 
 from mimesis.providers.base import BaseProvider
 
+import array
```

Comment:
1. standard library imports
2. related third party imports
3. local application/library specific imports

Reply:
I'll follow that convention from here on out then.